### DISCUSSION TITLE
Wireframes/ UML Designs Feedback

### DISCUSSION THREAD

**Discussion by noel-vargas** [uprm-inso4116-2024-2025-s1/semester-project-SafeRUM#66] [Ideas]
gHi, I created a drafts for: Profile Management Tab, UML Sequence Diagram for SafeRum Login and Account Creation, Forgot Password  Wireframes and I would love to hear some feedback. 

Account Creation:
![image](https://github.com/user-attachments/assets/4db2591f-fc43-4070-b96a-1da95241d153)

My Profile page:
![image](https://github.com/user-attachments/assets/316c5e9a-4d15-4fc4-bbe8-8667cde937f8)

Forgot Password Wireframes:
![image](https://github.com/user-attachments/assets/723e99c0-059e-49b2-b4d7-d0cb8b90bf1e)

UML Sequence Diagram:
![image](https://github.com/user-attachments/assets/6ac49705-fe73-4f22-b959-4f3108980725)

**Comment by shenied-maldonado** [comment]
Since the app is geared towards the UPRM community, I would say it should follow a color scheme that confides with the college's palette (white and green). I love the included logo and overall structure. Maybe we could fit the UPRM logo somewhere too! 

Here's the link to UPRM's brand resources and guidelines: https://www.uprm.edu/portales/marca/
Here's the UPRM identity manual: https://www.uprm.edu/wdt/resources/manual_identidad_grafica_upr-2014.pdf

Other than that, I know there's an open conversation on the font selection. I think that can be tweaked later on pretty easily. 

Good job!! ✨
